### ISSUE TITLE
Hyperspectra: pixel location plotting

### ISSUE THREAD

**Issue by stuart-cls** [markotoplak/orange-infrared#139]
_(I'm not sure if this is an issue that needs to be fixed, so this is more of a note of a possible issue in the future)_

In eafa7fb018dd26fc226071ee1b085aeef5e66519 the idea that pixel coordinates indicate the _center_ of the given pixel was introduced (_"shift centres of the pixels so that the axes are useful"_). At least in the case of the Agilent data, it would be more accurate to say that the pixel coordinates indicate the bottom-left corner of the pixel (i.e. 0,0 pixel should start at 0,0).

However, for the Agilent data it doesn't matter really since the locations are arbitrary and only the distances between pixels need to be correct (and they are).

What I'm unsure of is for OPUS/Omnic maps (at least) where the coordinates are real stage coordinates, this may be more of an issue. Again, for most analysis we are only interested in the size of things. 

To summarize, this is probably only a problem if we ever get visible image overlays working.

(I noticed this with the aggressively binned test data for the agilent reader if you want to see it)

**Comment by stuart-cls** [comment]
![pixel-centers](https://github.com/Quasars/orange-spectroscopy/assets/19228847/76f3c6ff-c36b-4560-9910-2daefa3665d6)

I was reminded of this bug adding tests for visible image loading in #726
